### DISCUSSION TITLE
Import demo sample data

### DISCUSSION THREAD

**Discussion by hitkoch** [pimcore/pimcore#15341] [Q&A / Support]
Hello, after days and many attempts, I finally managed to complete the installation on my VPS server and leave it active in the public url https://pimbr.com.br/admin
But now I have another problem, the demo content was not installed, please can someone help me and provide a SQL file so that I can import it into my database.
Tnks, Ronald Koch.

**Comment by mattamon** [comment]
Hey @hitkoch which project did you use to install pimcore?
https://github.com/pimcore/skeleton  -> the skeleton comes without any data.
https://github.com/pimcore/demo -> the demo installs the demo data in the install process.
In any case this does not work here are all the files:
Version 11 https://github.com/pimcore/demo/tree/11.x/dump
Version 10 https://github.com/pimcore/demo/tree/10.3/dump
